bin/test-merges.js

```diff
@@ -0,0 +1,296 @@
+#! /usr/bin/env node
+"use strict";
+
+/* eslint-disable no-console */
+/* this is a node script; primary interaction is via console */
+
+const Task = require("co-task");
+const process = require("process");
+const path = require("path");
+const GitHubApi = require("github");
+const shelljs = require("shelljs");
+
+const github = new GitHubApi({debug: false});
+
+// some of our API requests need to be authenticated
+let token = process.env.AS_PINE_TOKEN;
+github.authenticate({type: "token", token});
+
+// note that this token MUST have the public_repo scope set in the github API
+
+const AS_REPO_OWNER = process.env.AS_REPO_OWNER || "mozilla";
+const AS_REPO_NAME = process.env.AS_REPO_NAME || "activity-stream";
+const AS_REPO = `${AS_REPO_OWNER}/${AS_REPO_NAME}`;
+const OLDEST_PR_DATE = "2017-02-13";
```

Comment:
Actually, I'm inclined to just update it to today-ish, since I don't think people are likely to go groveling through the old PRs, and it would kick off a bunch of things the first time it runs as a cron job, I think.  Although I guess if that were true, it would have done that when you tested it this morning.  Still, it seems like a reasonable precaution, so I'm going to do that, and you can shout if you disagree.